File: src/main/java/vee/vaaw/util/ItemHelper.java
package vee.vaaw.util;

import vee.vaaw.Vaaw;
import vee.vaaw.item.*;
import vee.vaaw.item.group.VaawItemGroup;

public class ItemHelper {

    public static void initializeItems() {
        Vaaw.LOGGER.info("Initializing Items.");

        DevItems.INSTANCE.registerItems();
        WeaponItems.INSTANCE.registerItems();
        IngotItems.INSTANCE.registerItems();
        MaterialItems.INSTANCE.registerItems();
        MiscItems.INSTANCE.registerItems();

    }

    public static void initializeItemGroups() {
        Vaaw.LOGGER.info("Initializing Item Groups.");

        VaawItemGroup.registerGroup();
    }


}

File: src/main/kotlin/vee/vaaw/util/ItemHelper.kt
package vee.vaaw.util

import vee.vaaw.item.ArmourItems
import vee.vaaw.item.ConsumableItems
import vee.vaaw.item.IngotItems
import vee.vaaw.item.JokeItems
import vee.vaaw.item.MaterialItems
import vee.vaaw.item.TrinketItems
import vee.vaaw.item.WeaponItems
import vee.vaaw.item.group.ArmourItemGroup
import vee.vaaw.item.group.ConsumablesItemGroup
import vee.vaaw.item.group.JokeItemGroup
import vee.vaaw.item.group.MaterialsItemGroup
import vee.vaaw.item.group.TrinketItemGroup
import vee.vaaw.item.group.WeaponsItemGroup

object ItemHelper {

    fun initializeItems() {
        ArmourItems.registerItems()
        ConsumableItems.registerItems()
        IngotItems.registerItems()
        MaterialItems.registerItems()
        TrinketItems.registerItems()
        WeaponItems.registerItems()
        JokeItems.registerItems()
    }

    fun initializeItemGroups() {
        ArmourItemGroup.registerGroup()
        ConsumablesItemGroup.registerGroup()
        MaterialsItemGroup.registerGroup()
        TrinketItemGroup.registerGroup()
        WeaponsItemGroup.registerGroup()
        JokeItemGroup.registerGroup()
    }

}